common/src/components/accordion/Accordion.tsx

```diff
@@ -0,0 +1,101 @@
+import React, { useState } from "react";
+import { ChevronDownIcon, ChevronRightIcon } from "@heroicons/react/solid";
+
+export interface AccordionProps {
+  title: string;
+  subtitle?: string;
+  Icon?: React.ComponentType<React.SVGProps<SVGSVGElement>>;
+  context?: string;
+  simple?: boolean;
+  defaultOpen?: boolean;
+}
+
+interface InternalAccordionProps extends AccordionProps {
+  open?: boolean;
+}
+
+export const Accordion: React.FC<InternalAccordionProps> = ({
```

Comment:
Is InternalAccordionProps necessary now?
```suggestion
export const Accordion: React.FC<AccordionProps> = ({
```